### PR TITLE
c# file type maker

### PR DESCRIPTION
**Pull request by dannydulai**
support for c# using mono's mcs compiler -- same functionality as syntastic

### CODE REVIEW

File: autoload/neomake/makers/ft/cs.vim [hunk 1]
```diff
@@ -0,0 +1,14 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#cs#EnabledMakers()
+    if neomake#utils#Exists('mcs')
```
Comment (benekastah) [inline autoload/neomake/makers/ft/cs.vim]: You can remove this check. It's only needed in special circumstances. In other words, just returning `['mcs']` from this function unconditionally is fine.

**Comment by dannydulai** [comment]
@benekastah : done!

**Comment by benekastah** [comment]
Thanks!

**Comment by khalidchawtany** [comment]
@dannydulai how can I use **neomake** as a replacement to **syntastics**?

Currently I have the follwing to make **Syntastics** work?
```
 Plug 'scrooloose/syntastic'
 let g:syntastic_always_populate_loc_list = 1
 let g:syntastic_check_on_open = 1
 let g:syntastic_error_symbol = "✗"
 let g:syntastic_warning_symbol = "⚠"

 autocmd BufEnter,TextChanged,InsertLeave *.cs,*.cshtml SyntasticCheck

 Plug 'nosami/Omnisharp'
 let g:syntastic_cs_checkers = ['syntax', 'semantic', 'issues']
 setlocal omnifunc=OmniSharp#Complete


```

Note: I am on **OS X**. I use **Mono**.

Thanks.

**Pull request merged**